src/dynamics/video_player/player/states.js

```diff
@@ -867,6 +874,12 @@ Scoped.define("module:VideoPlayer.Dynamics.PlayerStates.PlayAd", [
             next: function(state) {
                 this.dyn.set("playing_ad", false);
                 inherited.next.call(this, state);
+            },
+
+            pause: function() {
+                this.dyn.player.pause();
+                // Fid ads player container to the content player container
+                this.dyn.trigger("resize");
```

Comment:
Hi @sambua, we shouldn't manually trigger `resize`, this event should only be triggered by the ResizeObserver.

```suggestion
            }
```